lib/DataRepeater.js

```diff
@@ -463,6 +463,8 @@ var DataRepeater = module.exports = kind(
 		// Decorate native JS array with at() so that we can
 		// access members of our dataset consistently, regardless
 		// of whether our data is in an array or a Collection
+		// TODO: Document that we add a method to an array or use
```

Comment:
Rather than documenting this, let's just fix it.

`Object.defineProperty(c, 'at', {value: at, enumerable: false})`